web/docs/guides/platform/permissions.mdx

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Seems like a good direction, but perhaps we will need something for them to rectify the permissions if they break them? If not, what is the use of this article - is this knowledge going to solve a problem for our users?

Reply:
yes? re-assigning ownership is not something a beginner's going to be doing, in which case, including a simple statement "alter ... owner to .." seems very low value to add, considering they're likely going to need to unwind other things they might've done on top of the ownership reassignment?

This article basically tells you, "yup, if you did this and you're seeing errors of this shape, that's expected and you need to get it back to this state"

Reply:
I think this is a good first step, even if just for a temporary measure. We can progressively make these requirements more user-friendly:

(Now) A section on the docs acknowledging potential permissions issues
-> A script users can run on the SQL editor as a "health check" for permissions issues (ala https://github.com/supabase/pgtap-poc)
-> Make that script a button on the dashboard
-> Run that script regularly on the background and report the results on a Lighthouse-like section on the dashboard